base/process.jl

```diff
@@ -72,6 +72,13 @@ backticks, e.g.
 """
 Cmd
 
+hash(x::Cmd) = hash(x.exec)
```

Comment:
let me know what you guys think of this. Without overriding the hash function, ```hash(``)``` would continually return different results which made ```Set``` not work properly so I think it makes sense to override the hash function but wasn't sure what a reasonable override was